app/javascript/chat/videoContent.jsx

```diff
@@ -1,55 +1,69 @@
-import { h, Component } from 'preact';
+import { h } from 'preact';
 
-export default class VideoContent extends Component {
-  render() {
-    if (!this.props.videoPath) {
-      return '';
-    }
+const smartSvgIcon = (content, d) => (
+  <svg
+    data-content={content}
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24"
+    width="24"
+    height="24"
+  >
+    <path data-content={content} fill="none" d="M0 0h24v24H0z" />
+    <path data-content={content} d={d} />
+  </svg>
+);
 
-    const smartSvgIcon = (content, d) => (
-      <svg
-        data-content={content}
-        xmlns="http://www.w3.org/2000/svg"
-        viewBox="0 0 24 24"
-        width="24"
-        height="24"
+/**
+ * Disaplys video in Connect.
+ *
+ * @param {string} props.videoPath The URL of a video
+ * @param {function} props.onTriggerVideoContent Fires when a video is clicked on
+ * @param {boolean} props.fullscreen Whether or not to show the video in fullscreen
+ *
+ * @returns A video component
+ */
+export function VideoContent(props) {
+  const { videoPath, onTriggerVideoContent, fullscreen } = props;
```

Comment:
The actual bug was the `fullscreen` prop not being used. It must have been destructured in another PR and somehow that got removed leaving the bare `fullscreen` var as undefined.